crates/testlang/outputs/cargo/crate/Cargo.toml

```diff
@@ -0,0 +1,54 @@
+[package]
+version.workspace = true
+rust-version.workspace = true
+edition.workspace = true
+publish = true
```

Comment:
I suggest removing the `publish = true`, so that it is ingored in CD.

Reply:
Done